module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,33 @@
+#include "vowel.hpp"
+
+std::vector<char> vowels = {'a', 'e', 'i', 'o', 'u', 'y', 'A', 'E', 'I', 'O', 'U', 'Y'};
+
+bool isVowel(char &character)
+{
+    if (std::find(vowels.begin(), vowels.end(), character) != vowels.end())
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
```

Comment:
This if/else is not necessary you can use `return std::find(vowels.begin(), vowels.end(), character) != vowels.end()`

Reply:
oh you are right.... i should've noticed that :p